### ISSUE TITLE
Feature request: match gSnap UI, Colors, and Animations with Gnome Theme

### ISSUE THREAD

**Issue by domferr** [GnomeSnapExtensions/gSnap#64]
I would like to suggest a new feature for gSnap. To improve the extension's integration with the currently selected Gnome theme, it would be great if the extension's UI, colors, and animations matched those of the theme.
For comparison, this is gSnap's UI:

![ui_comparison](https://github.com/GnomeSnapExtensions/gSnap/assets/14203981/41e35544-d09d-4406-9cb3-d288706aaa3e)

And this image shows how gSnap would appear if its UI and colors were matched to the Gnome theme:

![ui_matches_gnome_theme](https://github.com/GnomeSnapExtensions/gSnap/assets/14203981/40abdfa0-3de8-47fe-ade4-3e38bb161186)

As you can see from the last picture, I was already able to implement this feature by giving gSnap's zone the same CSS class as the Gnome theme's tiles (`tile-preview`). I've also added a bit of code to match Gnome's animations during zone selection. Here's a video that demonstrates the feature in action:

[ui_matches_gnome_theme-00.00.05.570-00.00.12.898.webm](https://github.com/GnomeSnapExtensions/gSnap/assets/14203981/cb24ba43-d2db-4077-8e7d-076504a5bf07)

If you think this feature would be useful, I can further refine my code and make everything optional from the settings. I'd be happy to open a pull request to submit my changes. Please let me know your thoughts!

**Comment by meronz** [comment]
Absolutely, you're always free to open PRs and suggest new features!

Personally, I like this feature and I could even think of it as the default however, from the video you posted It seems that the zones are very faintly visible even on a simple background like the one you have. It requires some experimentation before making it the default IMHO.

As always, I'll be happy to review/help with issues and code, though my time at the moment is limited, so expect a bit of a delay.

**Comment by domferr** [comment]
Hello, thank you for your reply. I'm happy that you like this feature and I agree with you that it should be the default however. I would like to open a pull request, but I made my changes on top of #61 and there is something incompatible with the current main branch. If you are thinking about merging #61 I would wait before opening a pull request, otherwise I can try to replicate the same behaviour by starting from the main branch.

Speaking about the zones color, the one that you see on my videos is exactly the same one that is used by the gnome's theme I am using right now. The changes I made basically set to each zone the same CSS class.
Probably it would be great to have as a default color the same as the gnome theme's color and then allow customizing the color in the preferences. Let me know what you think about!
